src/main/webapp/app/admin/system-notification-management/system-notification-management-detail.component.html

```diff
@@ -0,0 +1,23 @@
+<div *ngIf="notification">
+    <h2>
+        <span jhiTranslate="artemisApp.systemNotification.systemNotification">System notification</span> [<b>{{ notification.id }}</b
+        >]
+    </h2>
+    <dl class="row-md jh-entity-details">
+        <dt><span jhiTranslate="global.field.id">Id</span></dt>
+        <dd>
+            <span>{{ notification.id }}</span>
+        </dd>
+        <dt><span jhiTranslate="artemisApp.systemNotification.title">Title</span></dt>
+        <dd>{{ notification.title }}</dd>
+        <dt><span jhiTranslate="artemisApp.systemNotification.text">Text</span></dt>
+        <dd>{{ notification.text }}</dd>
+        <dt><span jhiTranslate="artemisApp.systemNotification.type">Type</span></dt>
+        <dd>{{ notification.type }}</dd>
+        <dt><span jhiTranslate="artemisApp.systemNotification.expireDate">Expire Date</span></dt>
+        <dd>{{ notification.expireDate | date: 'yyyy-MM-dd HH:mm' }}</dd>
```

Comment:
Is this date format wanted here? Maybe it makes sense in this scenario but I think in general we wanted to go with `dd-MM-yyyy HH:mm` everywhere on the platform.

Reply:
I just took it from the existing implementation of notification-component.html. I will adjust it in both places.

Reply:
I will use `dd.MM.yyyy HH:mm` as it seems to be the most common format in the code base.